graylog2-server/src/main/java/org/graylog/security/shares/EntityShareResponse.java

```diff
@@ -58,13 +58,18 @@ public abstract class EntityShareResponse {
     @JsonProperty("missing_permissions_on_dependencies")
     public abstract ImmutableMap<GRN, Collection<EntityDescriptor>> missingPermissionsOnDependencies();
 
+    @JsonProperty("synced_dependent_entities")
+    public abstract ImmutableSet<GRN> syncedDependentEntities();
```

Comment:
Let's remove the dependency terminology to avoid confusion.

```suggestion
    @JsonProperty("synced_entities")
    public abstract ImmutableSet<GRN> syncedEntities();
```